UPGRADE.md

```diff
@@ -2,6 +2,11 @@
 
 ## dev-develop
 
+### New security permissino for cache
+ 
+To be able to clear the cache the user need the permission LIVE in the
+webspace context.
```

Comment:
thats a BC break because if you dont do that you dont see something which you saw before. so it should be written here.